migrations/20220221155611_fixed_on_delete_for_other_related_tables/migration.sql

```diff
@@ -0,0 +1,17 @@
+-- DropForeignKey
```

Comment:
If there's no update of `schema.prisma` then we should not have any new data migrations

Reply:
Done.